Source/Core/Core/HW/DVD/DVDThread.cpp

```diff
@@ -219,11 +219,11 @@ bool UpdateRunningGameMetadata(const DiscIO::Partition& partition, std::optional
 
   if (title_id)
   {
-    u64 volume_title_id;
-    if (!s_disc->GetTitleID(&volume_title_id, partition))
+    const std::optional<u64> volume_title_id = s_disc->GetTitleID(partition);
+    if (!volume_title_id)
```

Comment:
It would probably be appropriate to shorten this to `if (!volume_title_id || *volume_title_id != *title_id)`, since `!volume_title_id` is a simple check compared to `!s_disc->GetTitleID(&volume_title_id, partition)`.